cpu/cortexm_common/vectors_cortexm.c

```diff
@@ -197,12 +197,15 @@ void reset_handler_default(void)
     dbgpin_init();
 #endif
 
+#ifndef DISABLE_CPU_INIT
```

Comment:
```suggestion
#if !DISABLE_CPU_INIT
```

I wonder if this would be better, given that the doc says

```
#define DISABLE_CPU_INIT 0
```

But defining it to `0` would counter-intuitively turn the disabling on.